Rule: any-kind-of-sankhara-at-all-you-should-see — 1 change(s), 1 file(s)

--- sujato/sutta/sn/sn12/sn12.70_translation-en-sujato.json
+++ sujato/sutta/sn/sn12/sn12.70_translation-en-sujato.json
@@ -1,2 +1,2 @@ sn12.70:16.4
 PLI: ye keci saṅkhārā atītānāgatapaccuppannā ajjhattaṁ vā bahiddhā vā oḷārikā vā sukhumā vā hīnā vā paṇītā vā ye dūre santike vā, sabbe saṅkhārā netaṁ mama nesohamasmi na meso attāti; evametaṁ yathābhūtaṁ sammappaññāya daṭṭhabbaṁ. 
-You should truly see any kind of saṅkhāras at all—past, future, or present; internal or external; solid or subtle; inferior or superior; far or near: <em>all</em> saṅkhāras—with right understanding: ‘This is not mine, I am not this, this is not my self.’ 
+You should truly see any kind of saṅkhāra at all—past, future, or present; internal or external; solid or subtle; inferior or superior; far or near: <em>all</em> saṅkhāras—with right understanding: ‘This is not mine, I am not this, this is not my self.’ 
